### ISSUE TITLE
Not working in Ubuntu 16.04

### ISSUE THREAD

**Issue by default50** [awslabs/ec2-migrate-longer-id#5]
In Ubuntu 16.04 I did:

```
git clone https://github.com/awslabs/ec2-migrate-longer-id.git
apt-get install python-virtualenv
virtualenv -p /usr/bin/python2.7 venv
source venv/bin/activate
pip install boto3
cd ec2-migrate-longer-id.git
./migratelongerids.py --status
```

And I got:

```
Traceback (most recent call last):
  File "./migratelongerids.py", line 22, in <module>
    from queue import Queue
ImportError: No module named queue
```

**Comment by default50** [comment]
This is fixed by PR #4 

I'm leaving the issue open until the PR is merged.

**Comment by brete** [comment]
This has now been merged in. The instructions are now fixed in the README.md file.
